aio_pika/tools.py

```diff
@@ -1,19 +1,20 @@
 import asyncio
 import logging
-from collections.abc import Set
 from functools import wraps
 from itertools import chain
 from threading import Lock
```

Comment:
Threading lock guarantees the execution in one thread. You can execute a callback in another thread or run an event-loop in another thread than the main.

In a normal situation, this lock is always released and does not lock anything, but ensures correct internal object states.

Reply:
@mosquito I don't quite understand why `asyncio.Lock` can't be used instead? If user needs to run something in another thread then it can be done with `asyncio.to_thread`.